repo/packages/D/dcos-aws-cli/0/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-aws-cli",
+  "version": "1.0",
+  "maintainer": "joerg@mesosphere.io",
+  "minDcosReleaseVersion": "1.9",
+  "description": "DC/OS AWS CLI",
```

Comment:
Add `The DC/OS AWS CLI simplifies some operations for cluster running on AWS. At this point, it only supports one command - listing public IPs.`